packages/backend/src/assets/ai.json

```diff
@@ -80,9 +85,10 @@
       "basedir": "recipes/natural_language_processing/chatbot-java-quarkus",
       "readme": "# Java-based chatbot application\n\nThis application implements a simple chatbot backed by Quarkus and its\nLangChain4j extension. The UI communicates with the backend application via\nweb sockets and the backend uses the OpenAI API to talk to the model served\nby Podman AI Lab.\n\nDocumentation for Quarkus+LangChain4j can be found at\nhttps://docs.quarkiverse.io/quarkus-langchain4j/dev/.",
       "recommended": [
+        "hf.bartowski.granite-3.1-8b-instruct-GGUF",
```

Comment:
question: why do change the provider to hf.bartowski over https://huggingface.co/lmstudio-community/granite-3.1-8b-instruct-GGUF ?

Reply:
This was the model given by management and the lmstudio one is based on it